config/routes.rb

```diff
@@ -13,10 +13,16 @@
     resources :deploy_groups, only: [:index]
 
     resources :projects, only: [:index] do
+      resources :stages, only: [:index] do
+        member do
```

Comment:
Yeah, kind of worried about that, I don't want to do three levels of resources foo.